### PR TITLE
UAG-375 Update Gutenberg template library

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- Card - https://ua-gutenberg.atlassian.net/browse/UAG-375

### Screenshots

- Issue - https://d.pr/Yilh4g

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Switch to master open a customizer and check the console you will get the react console error from main.js
- Switch to `update-gutenberg-template-library` and refresh the customizer issue resolved.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: lib/gutenberg-templates/classes/class-ast-block-templates.php [hunk 1]
```diff
@@ -390,7 +391,7 @@ public function template_assets() {
 						'wpforms_status'          => $this->get_plugin_status( 'wpforms-lite/wpforms.php' ),
 						'gutenberg_status'        => $this->get_plugin_status( 'gutenberg/gutenberg.php' ),
 						'_ajax_nonce'             => wp_create_nonce( 'ast-block-templates-ajax-nonce' ),
-						'button_text'             => esc_html__( 'Starter Templates', 'ultimate-addons-for-gutenberg' ),
+						'button_text'             => esc_html__( 'Starter Templates', 'ast-block-templates' ),
```
Comment (RajkiranBagal) [inline lib/gutenberg-templates/classes/class-ast-block-templates.php:394]: Please undo this change.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: lib/gutenberg-templates/classes/class-ast-block-templates.php (hunk 1 above)
Comment (AbhijitNage123) [reply lib/gutenberg-templates/classes/class-ast-block-templates.php:394]: It will update automatic when we creating a final PR.

**Review by RajkiranBagal — APPROVED** [review]

File: composer.lock [hunk 2]
```diff
@@ -50,16 +50,16 @@
         },
         {
             "name": "brainstormforce/gutenberg-templates",
-            "version": "1.0.8",
+            "version": "1.0.9",
```
Comment (sandeshjangam) [inline composer.lock:53]: Update version in composer.json file also.

**Review by sandeshjangam — APPROVED** [review]
